date.go

```diff
@@ -138,7 +139,16 @@ func (dst *Date) DecodeText(ci *ConnInfo, src []byte) error {
 	case "-infinity":
 		*dst = Date{Status: Present, InfinityModifier: -Infinity}
 	default:
-		t, err := time.ParseInLocation("2006-01-02", sbuf, time.UTC)
+		if sbuf[len(sbuf)-2:] == "BC" {
+			t, err := time.ParseInLocation("2006-01-02", strings.TrimRight(sbuf, " BC"), time.UTC)
+			t2 := time.Date(-t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second(), t.Nanosecond(), t.Location())
```

Comment:
zero year does not exist, so 1BC is effectively 1AD - 1 year

thus, you need to use: `1 - -t.Year()` here

Reply:
ok fixed